examples/differential-expression.md

```diff
@@ -1,23 +1,26 @@
 Find genes differentially expressed by the kidney at significance level 0.05:
+
 ```python
 >>> from os import environ
 >>> from hubmap_api_py_client import Client
 >>> client = Client(environ['API_ENDPOINT'])
 
->>> organ_name = client.select_organs().get_list()[1]['grouping_name']
->>> organ_genes = client.select_genes(where='organ', has=[organ_name], genomic_modality='rna', p_value=0.05)
->>> organ_genes_details = organ_genes.get_list()[0:10]
->>> organ_genes_details[0].keys()
-dict_keys(['gene_symbol', 'go_terms'])
+>>> modality = 'atac'
+>>> gene_symbol = client.select_genes(where='modality', has=[modality]).get_list()[10]['gene_symbol']
+>>> organs_with_gene = client.select_organs(where='gene', has=[gene_symbol], genomic_modality=modality, p_value=1.0)
+>>> organs_with_gene_details = organs_with_gene.get_list()[0:10]
+>>> organs_with_gene_details[0].keys()
+dict_keys(['grouping_name'])
```

Comment:
We've gone from `['gene_symbol', 'go_terms']` to `['grouping_name']`, which is a lot more generic: Is a `grouping_name` purely internal, or will it correspond to some vocabulary users will know?

Reply:
oh: looking at the code below, it looks like grouping name is just an organ? Will it always be organ?

Reply:
Behind the scenes, Datasets, Organs, Clusters, and soon Cell Types, are all subclasses of a class called Cell Grouping.  grouping_name refers generically to identifiers for any of these types of Cell Grouping.

Reply:
Please explain this in the documentation: If I'm confused in during the review, it's likely that any other user would also be confused, and they aren't going to dig through old PR comments.